appserver/payara-appserver-modules/microprofile/healthcheck/src/main/java/fish/payara/microprofile/healthcheck/admin/SetMPHealthCheckConfiguration.java

```diff
@@ -133,6 +157,46 @@ public void execute(AdminCommandContext context) {
         } catch (TransactionFailure ex) {
             actionReport.failure(LOGGER, "Failed to update HealthCheck configuration", ex);
         }
+
+        // If everything has passed, scrap the subaction reports as we don't want to print them out
+        if (!actionReport.hasFailures() || !actionReport.hasWarnings()) {
+            actionReport.getSubActionsReport().clear();
+        }
+    }
+
+    private boolean defaultMicroprofileUserExists(ActionReport subActionReport, Subject subject) {
+        boolean exists = false;
+
+        CommandRunner.CommandInvocation invocation = commandRunner.getCommandInvocation("list-file-users", subActionReport, subject,
+                false);
+
+        ParameterMap parameters = new ParameterMap();
+        parameters.add("authrealmname", "file");
+
+        invocation.parameters(parameters).execute();
+
+        for (ActionReport.MessagePart message : subActionReport.getTopMessagePart().getChildren()) {
+            if (message.getMessage().equals(DEFAULT_USER_NAME)) {
+                exists = true;
+                break;
```

Comment:
Not a big fan of `break` in loops as control flow is harder to follow then the alternative of doing a `return true` here and a `return false` at the end of the method instead.